packages/amplify-category-notifications/__tests__/lib/channel-FCM.test.js

```diff
@@ -0,0 +1,121 @@
+const inquirer = require('inquirer');
+const mockirer = require('mockirer');
```

Comment:
I guess this isn't a maitained package: https://preview.npmjs.com/package/mockirer.

Can we do like this : 

    jest.mock('inquirer', () => ({
      disableChannel: selectDisableChannel,
    }));
     selectDisableChannel.mockReturnValue(true)